src/NegativeFloat.js

```diff
@@ -0,0 +1,37 @@
+import { GraphQLScalarType } from 'graphql';
+import { GraphQLError } from 'graphql/error';
+import { Kind } from 'graphql/language';
+
+function processValue(value) {
+  if (isNaN(value)) {
```

Comment:
Unsure if this check is for detecting if the value passed is only a valid `number` or testing for `NaN`. Making an assumption based on the error message, maybe [Number.isFinite(value)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) would be better suited to validate that the value is not a Number. Assuming that you wish to disallow `NaN` & `Infinity`. 

Would recommend the changes for the other Number scalar types if the above assumption is correct.

Reply:
I'll check `isFinite`. I may also want to use `Number.isNaN()`. Thanks.

Reply:
There are some other things I can do to maybe make this more robust like checking against `Number.MAX_SAFE_INTEGER`, etc. for example.

Reply:
@ccuilla I realised that I should check the reference implementation as the `Number.isFinite()` will fail for strings. Check out how graphq.js are checking & coercing to [numbers](https://github.com/graphql/graphql-js/blob/master/src/type/scalars.js#L13).